src/app/profile/settings/connected-accounts/connected-accounts.component.html

```diff
@@ -35,8 +35,14 @@ <h2 class="card-pf-title">Connected Accounts</h2>
             <div class="col-xs-2">
               <img class="icon connected-account-icon" src="../../../../assets/images/openshift_icon.svg">
             </div>
-            <div class="account-text-box col-xs-10">
+            <div class="account-text-box col-xs-6">
               <span class="account-title">OpenShift</span>
+              <a [href]="cluster"
+                [tooltip]="cluster"
```

Comment:
@andrewazores `https://api.starter-us-east-2.openshift.com/` is what i get with user profile data as cluster URL. I can query `/user/services` API to get `https://console.starter-us-east-2.openshift.com/console/` as console URL but i don't like the idea of calling a separate API just to get one URL which avoids a simple redirect. 
As a workaround i can create console URLs using cluster name but that might break in future if URL structure changes.
@sbose78 What do you think we should do here?

Reply:
We shouldn't depend on the redirect. It might not always work with different clusters. 

You could skip calling user API altogether and call /API/user/services and display it here. 

In fact, in future, all of users OpenShift cluster info would be available in that endpoint probably.

Reply:
@rohitkrai03 is this being worked on further?

Reply:
@andrewazores Yes, i will use the new API to get cluster URL. My idea was to actually change `/user` auth endpoint so that it sends correct cluster URL along with user attributes but it will take some time. So after discussing within the team I've decided to use new endpoint `/user/services`. I will make changes to the PR tomorrow.